apps/native-component-list/src/screens/SensorScreen.tsx

```diff
@@ -180,6 +181,16 @@ class BarometerSensor extends SensorBlock<Sensors.BarometerMeasurement> {
   );
 }
 
+class LightSensor extends SensorBlock<Sensors.LightSensorMeasurement> {
+  getName = () => 'LightSensor';
+  getSensor = () => Sensors.LightSensor;
```

Comment:
Is this going to work on iOS without errors? Shouldn't we render this component only on Android?

Reply:
The sensors NCL screen loads only sensors for which `isAvailableAsync() == true`
So it isn't displayed on iOS, web and some Android phones without such sensor (tested it)